tests/checkbox/test-01-navigate-to-unchecked-checkbox-reading.html

```diff
@@ -6,18 +6,81 @@
 <link rel="help" href="https://w3c.github.io/aria/#checkbox">
 <link rel="help" href="https://w3c.github.io/aria/#aria-checked">
 
-<script src="scripts.js"></script>
+<script>
+  var scripts = {
```

Comment:
this is a bit strange, it might be nice to not include it if it's not necessary. but because they are generate files I suppose it doesn't matter, and this is not blocking.